src/data/wallets/wallet-data.ts

```diff
@@ -1593,26 +1593,26 @@ const walletData: WalletData[] = [
     macOS: true,
     firefox: false,
     chromium: true,
-    hardware: false,
+    hardware: true,
```

Comment:
TokenPocket isn't a hardware wallet. It has `hardware_supported` already (see line 1602)

```suggestion
    hardware: false,
```

Reply:
Ok, actually TokenPocket has its own hardware wallet, not named as TokenPocket though.